yaml_test.go

```diff
@@ -428,6 +428,7 @@ foo: baz
 }
 
 func TestJSONObjectToYAMLObject(t *testing.T) {
+	const bigUint64 = ((uint64(1) << 63) + 500) / 1000 * 1000
```

Comment:
you should not be touching yaml_test.go in your PR @longwuyuan please drop the commit that does this.